File: be/DTOs/VolunteerActionDTO.cs
namespace MealForFamily.DTOs
{
    public class VolunteerActionDTO
    {
        public int Id { get; set; }
        public VolunteerActionTypeDTO? Type { get; set; }
        public string? ImageURL { get; set; } = string.Empty;
        public string? Title { get; set; } = string.Empty;
        public VolunteerActionStatusDTO? Status { get; set; }
        public string? ShortDescription { get; set; } = string.Empty;
        public string? RawDescription { get; set; } = string.Empty;
        public string? Description { get; set; } = string.Empty;
        public string? ReferenceNumber { get; set; } = string.Empty;
    }
}

File: mob/src/models/VolunteerAction/VolunteerActionDTO.ts
export interface VolunteerActionDTO {
  id: number;
  type: ActionType;
  imageURL?: string;
  title: string;
  status: ActionStatus;
  shortDescription: string;
  referenceNumber?: string;
  description: string;
}

export interface ActionType {
  id: number;
  name: string;
}

interface ActionStatus {
  id: number;
  name: string;
}
